libs/design/scss/theming/contrast/max-contrast/max-contrast.scss

```diff
@@ -0,0 +1,40 @@
+@use 'sass:list';
+@use '../contrast-ratio/contrast-ratio.scss' as cr;
+@use '../../../core/error/error-to-string';
+
+/// Returns the color from a list of colors that has the highest contrast ratio with a reference color.
+/// If the list is empty, an error is thrown. If the list has one color, that color is returned.
+/// Uses the cr.daff-contrast-ratio function to calculate contrast ratios.
```

Comment:
I don't think `Uses the cr.daff-contrast-ratio function to calculate contrast ratios.` needs to be included in the docs.